docker/development-easy-light/docker-compose.yml

```diff
@@ -40,6 +40,7 @@ services:
     - vendordir:/var/www/localhost/htdocs/openemr/vendor:rw
     - ccdanodemodules:/var/www/localhost/htdocs/openemr/ccdaservice/node_modules:rw
     - logvolume:/var/log
+    - /couchdb/data
```

Comment:
I think you should not use an anonymous volume in this case, because people will want to retain and manage their couchdb data. I don't think that's the case with the .git data, based on how the development dockers are designed.